masterlist.yaml

```diff
@@ -1346,3 +1346,36 @@ plugins:
   - name: 'Convenient Horses.esp'
     after:
       - 'Open Cities Skyrim.esp'
+  - name: 'Book Covers Skyrim.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/901/'
+        name: 'Book Covers Skyrim on Nexus Mods'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/3071870/'
+        name: 'Book Covers Skyrim - Original on Bethesda.net'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/3072082'
+        name: 'Book Covers Skyrim - Desaturated on Bethesda.net'
+    after:
+      - 'UnlimitedBookshelves.esp'
+      - 'Weightless Books.esp'
+      - 'WeightlessOV - Book.esp'
+    tag:
+      - Graphics
+      - Names
+    clean:
+      # version 4.2
+      - crc: 0x32587221
+        util: SSEEdit v3.2.1
+  - name: 'Book Covers Skyrim - Lost Library.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/902/'
+        name: 'Book Covers Skyrim on Nexus Mods'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/3203152'
+        name: 'BCS Lost Library - Original on Bethesda.net'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/3203032'
+        name: 'BCS Lost Library - Desaturated on Bethesda.net'
+    tag:
+      - Delev
```

Comment:
Lost Library inserts books into Merchant and Item Levelled Lists.

Would it be possible to detect when a bash patch is required using tags? Or would it be just a guess?
I mean could loot cache crc for bash patch & check if a mod with Delev.Relev,Invent or Names was installed.